bims/static/js/app.js

```diff
@@ -156,6 +156,18 @@ require([
                 })
             }
             $('#general-info-modal').fadeOut();
-        })
+        });
+        $('#map-legend-wrapper').draggable({
```

Comment:
Please don't put the code here, put it on the olmap.js instead. We want to try to avoid cluttering up the app.js.

Reply:
done